openlibrary/plugins/openlibrary/js/scroll.js

```diff
@@ -0,0 +1,17 @@
+$(window).scroll(function() {
+    var height = $(window).scrollTop();
+    if (height > 200) {
+        $('#back2Top').fadeIn();
+    } else {
+        $('#back2Top').fadeOut();
+    }
+});
+$(document).ready(function() {
+    $("#back2Top").click(function(event) {
+        event.preventDefault();
```

Comment:
This is not needed since you return false below. Returning false in an event handler will stopPropagation and preventDefault.